app/Business/Password/Generator.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Business\Password;
+
+use Illuminate\Http\Request;
+
+class Generator
+{
+
+    public static function getPassword($request): string
```

Comment:
каждый метод должен быть обязательно с док.блоком.
Например тут:
```suggestion
    /**
    * @param \Illuminate\Http\Request $request
    *
    * @return string
    */
    public static function getPassword(Request$request): string
```